packages/migration/src/migrations/openhim/20240305193551-remove-openhim-collections.ts

```diff
@@ -9,29 +9,14 @@
  * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
  */
 
+import { dropCollectionsExcept } from '../../utils/resource-helper.js'
 import { Db, MongoClient } from 'mongodb'
 
 export const up = async (db: Db, client: MongoClient) => {
   const session = client.startSession()
   try {
     await session.withTransaction(async () => {
-      const collectionToKeep = 'changelog' as const
-      const allCollections = await db.listCollections().toArray()
-      const collectionNames = allCollections.map((col) => col.name)
-      const filteredCollectionNames = collectionNames.filter(
-        (c) => c !== collectionToKeep
-      )
-
-      for (const collection of filteredCollectionNames) {
-        await db
-          .collection(collection)
-          .drop()
-          .catch((error) => {
-            console.error(`Error dropping collection ${collection}:`, error)
-          })
-      }
-
-      console.log(`Removed collections: ${filteredCollectionNames.toString()}`)
+      await dropCollectionsExcept(db, 'changelog')
```

Comment:
You should duplicate the utility code to the migrations - they're specific to a point in time and if the utility was to change in the future it could break the old migrations